app/src/main/java/uk/ac/bath/petmatch/Database/Shelter.java

```diff
@@ -20,19 +20,22 @@
     private String gps;
     @DatabaseField
     private String email;
+    @DatabaseField
+    private String website;
 
     public Shelter() {
         // ORMLite needs a no-arg constructor
     }
 
-    public Shelter(String title, String description, String charityNumber, String address, String gps, String email) {
+    public Shelter(String title, String description, String charityNumber, String address, String gps, String email, String website) {
```

Comment:
You would need to run the DatabaseConfigUtil in order for this to work, I'll do that for you now